sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -215,10 +219,13 @@ export abstract class MessageReceiver extends LinkEntity<Receiver> {
    * You may want to set up retries to recover the broken link and/or report error to user.
    * @param error - The error accompanying the receiver/session error or connection disconnected events
    */
-  abstract onDetached(error?: AmqpError | Error): Promise<void>;
+  abstract onDetached(
+    error: AmqpError | Error | undefined,
+    operationOptions: OperationOptionsBase | undefined
+  ): Promise<void>;
 
   /**
-   * Clears lock renewal timers on all active messages, clears token remewal for current receiver,
+   * Clears lock renewal timers on all active messages, clears token renewal for current receiver,
```

Comment:
I think @ramya-rao-a  might have put this in as a cat joke. Or maybe it's a typo.